src/System.Net.Http/tests/FunctionalTests/HttpRequestMessageTest.cs

```diff
@@ -251,6 +251,35 @@ public async Task HttpRequest_BodylessMethod_NoContentLength(string method)
             }
         }
 
+        [Fact]
+        public async Task HttpRequest_BodylessMethod_LargeContentLength()
```

Comment:
I don't understand what this test method is verifying.  There are no "Assert.*" calls in the test.  So, is this a manual test because you are looking at memory consumption or something?

I think this test needs to be modified to actually measure something and verify it with an Assert.* call. Otherwise it isn't really testing anything other than validating that it doesn't hang or generate an exception.

Reply:
It is testing that it's not throwing an exception (as it would without this PR).